src/extended_navigation_controller.js

```diff
@@ -301,11 +301,24 @@ export class ExtendedNavigationController extends NavigationController {
   }
 }
 
+function addAriaInfo(workspace, navController) {
+  const injectionDiv = workspace.getInjectionDiv();
+  injectionDiv.setAttribute('role', 'application');
+  injectionDiv.setAttribute('aria-label', 'Blockly workspace');
+  injectionDiv.setAttribute('tabindex', '0');
+  // Set focus when entering the region.
+  injectionDiv.addEventListener('focusin', (event) => {
```

Comment:
Just a Q: How does this interact with 1) Clicking on the workspace (does that fire a focus event) 2) modifying text input fields, since the text input is outside the workspace div?

Reply:
1) clicking inside the workspace also fires a focus event, so it flips the cursor back to the top level of the workspace. I think this is part of the discussion in https://github.com/google/blockly-keyboard-experimentation/discussions/11

Reply:
2. This does not do what I want. Back to the drawing board.